backend/app/Main.hs

```diff
@@ -4,20 +4,16 @@ module Main
 
 import Universum
 
-import qualified Data.Yaml as Y
 import Main.Utf8 (withUtf8)
-import Options.Applicative (execParser, fullDesc, helper, info, progDesc)
+import Options.Applicative (execParser)
 
-import Edna.Config.Definition (defaultEdnaConfig)
-import Edna.Config.Utils (configPathParser)
-import Edna.Setup (runEdna)
+import Edna.Config.CLA (ednaOpts)
+import Edna.Setup (prepareConfig, runEdna)
 import Edna.Web.Server (edna)
 
 main :: IO ()
 main = withUtf8 $ do
+  options <- execParser ednaOpts
+  config <- prepareConfig options
   hPutStrLn @Text stderr "Edna server is started"
```

Comment:
I think it's better to put it before options parsing. This message is helpful for debugging (e. g. to debug Docker stuff) when you want to know whether the server was started at all. If we write it after parsing, there is a small chance that the application will die during parsing and the message won't be printed.

Reply:
the reason I've put it there - if we run Edna with `-h` option (to see help) we also will see message that Edna was started.. this is not correct

But we can put it one line upper, after options parsing and before configuration. What do you think?

Reply:
Okay, sounds like a good compromise, let's do it.